extras/ams_wrapper/src/api/models/classification_attributes_model.py

```diff
@@ -26,29 +26,28 @@
 logger = get_logger(__name__)
 
 
-class VehicleAttributes(Model):  
+class ClassificationAttributes(Model):  
 
     def postprocess_inference_output(self, inference_output: dict) -> str:
-
-        # model with output shape for color (1,7,1,1) 
-        # with second dimension containing colors
-        # [white, gray, yellow, red, green, blue, black]
-        # model with output shape for type (1,4,1,1) 
-        # with second dimension containing types
-        # [car, bus, truck, van]
-        outputs = self.labels
+        # model with output shape for each classification output_name (1,N,1,1) 
         classifications = []
-        for output in outputs.keys():
-            type_name = output
+        for output_name in self.labels.keys():
             attributes = []
             highest_prob = 0.0
-            for position in outputs[output].keys():
-                class_name = outputs[output][position]
-                probability = inference_output[type_name][0,int(float(position)),0,0].item()
+
+            if output_name not in inference_output:
+                message = 'Model configuration label- {}'
+                ' not found in model output - {}'.format(output_name, inference_output)
```

Comment:
It's a little unclear here in my opinion. Could we change this message to something like:
"Output name from model config does not match model outputs" ?